addons/event/models/event_registration.py

```diff
@@ -40,6 +40,8 @@ class EventRegistration(models.Model):
         readonly=False, store=True)
     event_begin_date = fields.Datetime(string="Event Start Date", related='event_id.date_begin', readonly=True)
     event_end_date = fields.Datetime(string="Event End Date", related='event_id.date_end', readonly=True)
+    event_organizer_id = fields.Many2one(string='Event organizer', related='event_id.organizer_id', readonly=True)
+    event_user_id = fields.Many2one(string='Event responsible', related='event_id.user_id', readonly=True)
```

Comment:
We usually capitalize those
("Event Organizer" and "Event Responsible").

Reply:
Done, thanks